docs/content/ActionList.mdx

```diff
@@ -3,7 +3,7 @@ componentId: action_list
 title: ActionList
 status: Beta
 source: https://github.com/primer/react/tree/main/src/ActionList
-storybook: '/react/storybook?path=/story/composite-components-actionlist'
+storybook: 'https://primer.style/react/storybook?path=/story/components-actionlist'
```

Comment:
Real quick, just wanted to ask about the switch to including `https://primer.style` versus `/` and what the pros/cons of each was 👀

Reply:
I am glad you pointed out @joshblack because actually this is a mistake, now I realised 🙈 

My intention switching to `https://primer.style` was because`/` doesn't direct us to the correct link on the dev environment. See [this](https://primer-cde231ddd0-13348165.drafts.github.io/ActionList) for example and click on [Storybook](https://primer-cde231ddd0-13348165.drafts.github.io/react/storybook?path=/story/composite-components-actionlist. I have seen a couple of components using `https://primer.style` and I jumped straight into it. And it was late last night when I was testing this, so totally missed the elephant here 😅 Apparently, this fixes broken links but redirects to `primer/style` not the dev env the branch is running on! I'll remove the `https://primer.style` now. Also I thought about removing `react` from the url and this seems to fix this problem but I am not sure if I miss anything here. Like why did we have `react` in the first place in the url..

Reply:
Apparently we need `react` because the domain name in prod is `primer.style` not `primer.style/react`! Ah this confused me. Anyhow, I reverted back to the original version and just updated story links that caused by the format change. 

There are still 3 components using `primer.style` as opposed to dynamic domain name, I haven't touched them but asked @colebemis about it.